scripts/scan_results.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+"""
+Script to fetch test status info from sqlit data base. Before use this
+script, avocado We must be lanuch with '--journal' option.
+"""
+
+import os
+import sys
+import sqlite3
+import argparse
+from dateutil import parser as dateparser
+
+
+def fetch_data(db_file=".journal.sqlite"):
+    records = []
+    con = sqlite3.connect(db_file)
+    try:
+        cur = con.cursor()
+        cur.execute("select tag, time, action, status  from test_journal")
+        while True:
+            # First record contation start info, second contain end info
+            # merged start info and end info into one record.
+            data = cur.fetchmany(2)
+            if not data:
+                break
+            tag = data[0][0]
+            result = "N/A"
+            status = "Running"
+            end_time = None
+            end_str = None
+            elapsed = None
+            start_time = dateparser.parse(data[0][1])
+            start_str = start_time.strftime("%Y-%m-%d %X")
+            if len(data) > 1:
+                status = "Finshed"
+                result = data[1][3]
+                end_time = dateparser.parse(data[1][1])
+                elapsed = (end_time - start_time).total_seconds()
+                end_str = end_time.strftime("%Y-%m-%d %X")
+            record = (tag, status, start_str, end_str, result, elapsed)
+            records.append(record)
+    finally:
+        con.close()
+    return records
+
+
+def print_data(records):
+    if not records:
+        return
+    print "%-40s %-15s %-15s %-15s %-10s %-10s" % (
+        "CaseName", "Status", "StartTime",
+        "EndTime", "Result", "TimeElapsed")
+    for row in records:
+        print "%s %s %s %s %s %s" % row
+
+
+if __name__ == "__main__":
+    default_results_dir = os.path.join(
+        os.path.expanduser('~'),
+        'avocado',
+        'job-results',
+        'latest')
```

Comment:
How about get the log directory path by using the function provided by avocado itself, since we may customlize it.

>>> from avocado.core import data_dir
>>> data_dir.get_logs_dir()
'/root/avocado/job-results'